packages/compass-connections-navigation/src/connections-navigation-tree.tsx

```diff
@@ -181,49 +166,8 @@ const ConnectionsNavigationTree: React.FunctionComponent<
           />
         )}
       </AutoSizer>
-    </>
-  );
-};
-
-const MCContainer = css({
-  display: 'flex',
-  flex: '1 0 auto',
-  height: `calc(100% - ${spacing[1600]}px - ${spacing[200]}px)`,
-});
-
-const SCContainer = css({
-  display: 'flex',
-  flex: '1 0 auto',
-  height: 0,
-});
-
-const contentContainer = css({
-  display: 'flex',
-  flex: '1 0 auto',
-});
-
-const NavigationWithPlaceholder: React.FunctionComponent<
-  { isReady: boolean } & React.ComponentProps<typeof ConnectionsNavigationTree>
-> = ({ isReady, ...props }) => {
-  const isSingleConnection = !usePreference(
-    'enableNewMultipleConnectionSystem'
-  );
-
-  return (
-    <FadeInPlaceholder
-      className={isSingleConnection ? SCContainer : MCContainer}
-      contentContainerProps={{ className: contentContainer }}
-      isContentReady={isReady}
-      content={() => {
-        return (
-          <ConnectionsNavigationTree {...props}></ConnectionsNavigationTree>
-        );
-      }}
-      fallback={() => {
-        return <TopPlaceholder />;
-      }}
-    ></FadeInPlaceholder>
+    </div>
   );
 };
```

Comment:
I removed this top level WithPlaceholder component because I noticed that both legacy sidebar and multi connections sidebar were always passing isReady as true to this component which led me thinking that it was not needed anymore.

Reply:
Are we still passing isReady or is that cleaned up too?

Reply:
yea I cleaned them up as well

Reply:
thank you!